python/HeaderService/hutils.py

```diff
@@ -33,6 +33,10 @@
 import itertools
 import copy
 from .camera_coords import CCDGeom
+import astropy
```

Comment:
The usual convention in Astropy is to do `from astropy.time import Time`

Reply:
Updated to `from astropy.time import Time` at the top of the file.